src/migrate.py

```diff
@@ -0,0 +1,49 @@
+from tv import app, db
+import os
+import re
+import importlib
+
+def ensure_migration_table():
+    with app.app_context():
+        table = db.session.execute("SELECT name FROM sqlite_master WHERE type='table' AND name='migration'")
+        if len(table.fetchall()) != 1:
+            print("Creating migration table")
+            db.session.execute("CREATE TABLE migration (version INTEGER)")
+            db.session.execute("INSERT INTO migration (version) VALUES (0)")
+        db.session.commit()
+
+def get_migration_version():
+    ensure_migration_table()
+    with app.app_context():
+        version = db.session.execute("SELECT version FROM migration").fetchall()[0][0];
```

Comment:
Based on the other code, this should work, but I would add a check if the return is more than just one line.

Reply:
mm, maybe there is a way to get sqlite to ensure a database only has one row

Reply:
seems nice maybe https://stackoverflow.com/a/33104119

Reply:
> seems nice maybe https://stackoverflow.com/a/33104119

now in codebase :p